### PR TITLE
axel: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. We can use long arguments here but IG the current one is fine.

File: pages.es/common/axel.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# axel
+
+> Acelerador de descargas.
+> Soporta HTTP, HTTPS y FTP.
```
Comment (realFranco) [inline pages.es/common/axel.md:4]: Hello, thank you for the contribution, I'll add a suggestion.

```suggestion
> Protocolos soportados HTTP, HTTPS y FTP.
```

**Review by realFranco — CHANGES_REQUESTED** [review]

File: pages.es/common/axel.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+> Más información: <https://github.com/axel-download-accelerator/axel>.
+
+- Descarga una URL a un archivo:
```
Comment (realFranco) [inline pages.es/common/axel.md:7]:
```suggestion
- Descarga un archivo alojado en una URL pública:
```

**Review by realFranco — CHANGES_REQUESTED** [review]

File: pages.es/common/axel.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Descarga y especifica nombre de archivo:
+
+`axel {{url}} -o {{ruta/al/archivo}}`
+
+- Descarga con varias conexiones:
+
+`axel -n {{num_conexiones}} {{url}}`
+
+- Busca copias espejo:
```
Comment (realFranco) [inline pages.es/common/axel.md:19]:
```suggestion
- Busca un archivo utilizando un mirror haciendo uso del motor de búsqueda filesearching.com.
```

**Review by realFranco — CHANGES_REQUESTED** [review]

File: pages.es/common/axel.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Descarga un archivo alojado en una URL pública:
+
+`axel {{url}}`
+
+- Descarga y especifica nombre de archivo:
+
+`axel {{url}} -o {{ruta/al/archivo}}`
+
+- Descarga con varias conexiones:
```
Comment (navarroaxel) [inline pages.es/common/axel.md:15]:
```suggestion
- Descarga con múltiples conexiones:
```

File: pages.es/common/axel.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+> Acelerador de descargas.
+> Protocolos soportados HTTP, HTTPS y FTP.
+> Más información: <https://github.com/axel-download-accelerator/axel>.
+
+- Descarga un archivo alojado en una URL pública:
+
+`axel {{url}}`
+
+- Descarga y especifica nombre de archivo:
```
Comment (navarroaxel) [inline pages.es/common/axel.md:11]:
```suggestion
- Descarga y especifica un nombre de archivo:
```

File: pages.es/common/axel.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# axel
+
+> Acelerador de descargas.
+> Protocolos soportados HTTP, HTTPS y FTP.
+> Más información: <https://github.com/axel-download-accelerator/axel>.
+
+- Descarga un archivo alojado en una URL pública:
```
Comment (navarroaxel) [inline pages.es/common/axel.md:7]:
```suggestion
- Descarga un archivo alojado en una URL:
```

File: pages.es/common/axel.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+- Descarga y especifica nombre de archivo:
+
+`axel {{url}} -o {{ruta/al/archivo}}`
+
+- Descarga con varias conexiones:
+
+`axel -n {{num_conexiones}} {{url}}`
+
+- Busca un archivo utilizando un mirror haciendo uso del motor de búsqueda filesearching.com.
```
Comment (navarroaxel) [inline pages.es/common/axel.md:19]: this is not a translation.

Comment (kant) [reply pages.es/common/axel.md]: @navarroaxel fixed

**Comment by kant** [comment]
[Codespell](https://github.com/tldr-pages/tldr/actions/workflows/codespell.yml)

**Review by blueskyson — APPROVED** [review]

**Pull request merged**